devbuild.sh

```diff
@@ -237,6 +237,9 @@ if [ "${BUILD_CONDA}" = "on" ] ; then
   if ! conda env list | grep -q "^srw_graphics\s" ; then
     mamba env create -n srw_graphics --file graphics_environment.yml
   fi
+  if ! conda env list | grep -q "DEV-uwtools" ; then
```

Comment:
```suggestion
  if ! conda list | grep -q "^condev\b"; then
```
It seems like what's wanted is "If condev is not installed, install it". The existence of a `DEV-uwtools` environment kind of implies that `condev` is installed, but I think this would be a more explicit test.